packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -779,6 +793,9 @@ export class List<T = any> extends BaseComponent<IListProps<T>, IListState<T>> i
 
         newPage.top = pageTop;
         newPage.height = pageHeight;
+        if (this._visibleRect && this._visibleRect.bottom) {
+          newPage.isVisible = pageBottom >= this._visibleRect.top && pageTop <= this._visibleRect.bottom;
```

Comment:
is this used anywhere?

Reply:
Yes, we need this for a feature in odsp.